source/faq.rst

```diff
@@ -52,10 +53,60 @@ How do I navigate to another Pod?
 To browse another Pod (or even a different Container in your own Pod), you can navigate there 
 directly by entering the Pod (or Container) URI.
 
-#. In PodBrowser, click the current :guilabel:`Pod` identifier link (just below the application logo).
+#. In PodBrowser, click the current :guilabel:`Pod` identifier link (just below the Account menu).
+
+#. Select :guilabel:`Change Pod` in the menu.
+
+#. In the displayed panel, enter the URI for the Pod (or Container).
+
+#. Press :guilabel:`Enter`, or click the :guilabel:`Go` button.
+
+#. The navigation panel is refreshed and shows the Resources in the entered Pod (or Container).
+
+What do the Resource permissions mean?
+======================================
+
+PodBrowser allows you to share Containers and Resources within your
+Pod with other Solid users. But what do the available permissions allow the other Solid users to do? 
+The table below describes the permissions which can be conbined together:
+
+.. list-table::
+  :header-rows: 1
+  :widths: 20 80
+
+  * - Permission
+
+    - Other Solid user can...
+
+  * - View
+
+    - View the Resources within a Container, or download a Resource.
+
+  * - Edit
+
+    - Change the contents of or delete a Resource.
+
+  * - Append
+
+    - Add to a Resource, but not change the existing contents.
+
+  * - Control
+
+    - Change the permissions for a Resource.
+      
+.. note:: To allow other Solid users to display the Resource details in PodBrowser, 
+  and hence access the Download and Delete options, they must have Read access to the Resource.
+
+
+What is stored in ``/pb_policies``?
+===================================
 
-#. In the displayed :guilabel:`Go to Pod` panel, enter the URI for the Pod (or Container).
+When you access a Pod stored on an `Enterprise Solid Server`_ (e.g., https://pod.inrupt.com) using PodBrowser, 
```

Comment:
Maybe flip the phrase so that we emphasize that it's PodBrowser specific:

PodBrowser creates a new root Container ``/pb_policies`` for Pods stored on an `Enterprise Solid Server`_ (e.g., ``https://pod.inrupt.com``). The ``/pb_policies`` stores the `Access Control Policies`_, or permissions`, for the Resources within the Pod.

(I would wrap https://pod.inrupt.com with the backticks ``... `` to avoid people linking out to the 401 page).

Reply:
OK

Reply:
Ended up hyperlinking https://pod.inrupt.com to https://signup.pod.inrupt.com.